docs/data/base/components/click-away-listener/click-away-listener.md

```diff
@@ -1,58 +1,59 @@
 ---
 product: base
-title: Detect click outside React component
+title: React ClickAwayListener component
 components: ClickAwayListener
 githubLabel: 'component: ClickAwayListener'
 packageName: '@mui/base'
 ---
 
 # Click-away listener
 
-<p class="description">Detect if a click event happened outside of an element. It listens for clicks that occur somewhere in the document.</p>
+<p class="description">The <code>ClickAwayListener</code> component detects when a click event happens outside of its children elements.</p>
 
-- 📦 [981 B gzipped](/size-snapshot).
+- 📦 [981 B gzipped](/size-snapshot)
 - ⚛️ Supports portals
```

Comment:
What does this mean? Can we elaborate or be more specific?

```suggestion
- ⚛️ Supports portals
```

Reply:
Portals are elements that are rendered outside of the parent's DOM tree. Technically it is a bit harder to implement click away listener because the event is not bubbling as you would except (by default). This is why this is considered as an additional feature, and there is an example showcasing that it works :)